### DISCUSSION TITLE
Pony Origins 1.3.0 Beta 2 for Origins 1.10.x

### DISCUSSION THREAD

**Discussion by BlasterM5** [BlasterM5/ponyorigins#13] [Announcements]
Pony Origins Update

This update fixes a couple bugs, and adds some more food mod compatibilities.

* Added support for Ocean's Delight
* Added support for Fright's Delight
* Added support for Bodacious Berries
* Fixed some tag entries
* Tweaked some power descriptions
* Reworked Teleport - Max range is now 200 blocks, Spatium Pontis 400 blocks
* Teleports now only consume the needed energy (within the nearest 10 rounded up)
* Emotivores can not get the Nourishment buff
* Juices count as "fruit meals" for Fructivores, providing the normal amount of saturation
* Web Shot no longer destroys doors by placing cobwebs in them
* Shadow Walk's particle effect is darker and more shadowey
* Compressed the jar file, reducing its size from 1.2 MB to just 220 KB

<hr /><em>This discussion was created from the release <a href='https://github.com/BlasterM5/ponyorigins/releases/tag/1.3.0-beta2'>Pony Origins 1.3.0 Beta 2 for Origins 1.10.x</a>.</em>

**Comment by AndreasMatian** [comment]
Hello, I wanted to ask you how to become an alicorn?
I became a master of all three races, but did not understand what to do next.
Do I need to get the "Make your mark" achievement?
If so, what do I need to do to complete it?

**Reply by BlasterM5** [reply]
Make Your Mark achievements trigger when you complete one of Minecraft's hardest achievements. Try defeating the Ender Dragon, perhaps?

**Reply by AndreasMatian** [reply]
Thanks for the answer and the cool mod!
